### PR TITLE
Fix: example manager and instructions

### LINKED ISSUE #47: Add api group to clusterrole's first rule in config/manager
**Describe the bug**
Getting the error
```
The ClusterRole "kube-trigger-manager-role" is invalid: rules[0].apiGroups: Required value: resource rules must supply at least one api group
```
when applying `config/manager/role.yaml` file.

**To Reproduce**
1. Follow the [Quick Start](https://github.com/kubevela/kube-trigger/blob/main/examples/README.md#quick-start) guide.
2. Apply all the files in `config/crd`, `config/definition`, and `config/manager` respectively.
3. `kubectl apply -f` will throw the error when trying to apply `role.yaml`.

**Expected behavior**
Successful creation of `ClusterRole` `kube-trigger-manager-role`

**Kube Trigger Version**
- kube-trigger version: `1.0.0` (kube-trigger addon)

**Cluster information**
- Minikube version: `v1.30.1`
- Kuberentes version: `v1.24.13`
- Kubevela version: `v1.8.1`

**Additional context**
I enabled the `kube-trigger` addon after installing `kubevela` .

### PR DESCRIPTION
**Pull request by semmet95**
### Description of your changes
Updated **ClusterRole** and **Quick Start** guide.
Fixes #47

Fixes #
- `rules[0].apiGroups: Required value: resource rules must supply at least one api group` error.

I have:
- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Add related tests.
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### How has this code been tested
By running the quick tutorial with the updated examples.

### Special notes for your reviewer
I have updated the **Quick Start** guide's **Prerequisites** section to including enabling the `kube-trigger` addon

### CODE REVIEW

**Comment by FogDong** [comment]
Please sign the DCO with `git commit -s`

File: config/manager/role.yaml [hunk 1]
```diff
@@ -5,7 +5,9 @@ metadata:
   creationTimestamp: null
   name: kube-trigger-manager-role
 rules:
-- resources:
+- apiGroups:
```
Comment (FogDong) [inline config/manager/role.yaml:8]: I think this file is auto generated, can you help to check that is there anything wrong with the generator? cc @charlie0129

**Review by FogDong — APPROVED** [review]

File: config/manager/role.yaml (hunk 1 above)
Comment (charlie0129) [reply config/manager/role.yaml:7]: Yes, that's right. Maybe newer versions of kubectl require apiGroups to be set.

I fixed the generator to generate that field.

**Review by charlie0129 — APPROVED** [review]
Great job!

Can you sign the DCO? There are instuctions if you click on `Details` on DCO check.

**Comment by semmet95** [comment]
Hey @FogDong and @charlie0129 
Thanks for approving, I'm trying to get the old commits signed, or will probably squash them all.

**Comment by semmet95** [comment]
Hey
So it seems like I made over a 100 commits to my branch and still couldn't get rid of the signed commit check errors 🥹.
Do you guys want me to create a new PR from a new branch with all these changes?

**Comment by charlie0129** [comment]
You can use force-pushing to get rid of the past (unsigned) commits. IMO, continue on this branch is fine.

**Comment by charlie0129** [comment]
You can use `git reset --hard 69bc6f6` to clean up all the irrelevant commits (make sure you have some kind of backup because it is a hard reset). Then `git push -f` to force push to your branch.

Then you can solve the DCO

**Comment by semmet95** [comment]
Thanks a lot @charlie0129 
The DCO check is finally passing now

**Comment by charlie0129** [comment]
The code changes doesn't seem right. It contains many unrelated changes.

**Comment by semmet95** [comment]
Can I reset back to commit [b76f491](https://github.com/kubevela/kube-trigger/pull/48/commits/b76f491ec3383df933f7d20de6f7e15dcc9b6bfe)?
That should remove the unrelated changes.

**Comment by semmet95** [comment]
@charlie0129 I've removed all the unrelated changes and all the commits are signed too.

**Comment by FogDong** [comment]
Great work! Thanks @semmet95 !

**Pull request merged**